test/sequential/test-repl-persistent-history.js

```diff
@@ -166,6 +172,18 @@ const tests = [{
   env: {},
   test: [UP],
   expected: [prompt, homedirErr, prompt, replDisabled, prompt]
+},
+{
+  env: { NODE_REPL_HISTORY: emptyHistoryPath,
+         NODE_REPL_HISTORY_FILE: emptyHistoryPath },
+  test: [UP],
+  expected: [prompt, sameHistoryFilePaths, prompt]
+},
+{
+  env: { NODE_REPL_HISTORY: defaultHistoryPath,
+         NODE_REPL_HISTORY_FILE: emptyHistoryPath },
+  test: [UP],
+  expected: [prompt, convertMsg, prompt]
```

Comment:
Please see the last test before this change. It has a comment stating it must be the last for very specific reasons and tests down here will have failures due to it.